### ISSUE TITLE
Bootstrap Package overrides

### ISSUE THREAD

**Issue by Tonyt8** [benjaminkott/bootstrap_package#331]
Dear Benjamin,
I have used bootstrap_package_overrides 0.0.1 in combination with bootstrap_package on some of my 7.6+ installations.
My question: Will bootstrap_package_overrides continue to work/function fully with new versions of the bootstrap_package?

Kind regards,
Tony

**Comment by cedricziel** [comment]
Hi @Tonyt8 ,

as I maintain `bootstrap_package_overrides` as an example on how to override specfic parts of the original bootstrap to show how decoupling both for profit in your own projects, [the `bootstrap_package_overrides` repository](https://github.com/cedricziel/bootstrap_package_overrides/issues) would have been a better place to ask.

As the overrides repository initially was just an example project it absolutely isn't up to date with the bootstrap package itself. However if your question ist "can I override specific parts of the package at will?", then the answer is yes. Of course.

Please be aware that there has been a lot of restructuring as the cms itself evolves constantly and so does our view on projects. You may have a rocky road ahead if you intend to update existing installations to the current version of bootstrap package though it can be done.

Cheers,
Cedric

**Comment by Tonyt8** [comment]
Hello Cedric,

Thanks for the rely.
You have confirmed my fears, that it may become difficult to continue using the bootstrap_package_overrides as it is.
I am a designer so I must take care what method I use to implement my own custom templates and css. 
I guess when bootstrap version 4+ is used the LESS theme I have used in the overrides will fail, is this correct?
Anyway a big thank you for making the overrides package. It has helped me understand some of the bootstrap/Typo3 methods. I have used it on 2 projects, but maybe it would be wise to use a simpler method in the future.

Are you planing to make any future updates to the bootstrap_package_overrides in the future?

Kind regards,
Tony
